### PR TITLE
Wip/ji/log database

### PR DESCRIPTION
**Pull request by jacek-izykowski**
Closed db view and log view can be reopened through "View" menu.

### CODE REVIEW

File: src/ui/veles_mainwindow.cc [hunk 1]
```diff
@@ -984,6 +984,8 @@ bool MainWindowWithDetachableDockWidgets::splitDockWidgetImpl(
 VelesMainWindow::VelesMainWindow() : MainWindowWithDetachableDockWidgets() {
   setAcceptDrops(true);
   resize(1024, 768);
+  database_dock_widget_ = 0;
+  log_dock_widget_ = 0;
```
Comment (altran01) [inline src/ui/veles_mainwindow.cc]: nit. we could use initializer lists instead

File: src/ui/veles_mainwindow.cc [hunk 2]
```diff
@@ -1114,9 +1128,27 @@ void VelesMainWindow::updateParsers(dbif::PInfoReply reply) {
   }
 }
 
+void VelesMainWindow::showDatabase() {
+  if (database_dock_widget_ == nullptr) {
+    createDb();
+  }
+
+  database_dock_widget_->raise();
```
Comment (altran01) [inline src/ui/veles_mainwindow.cc]: Hmm, one thing that might be slightly confusing about this implementation is that if database is in other main window (hidden under current one) user won't see any change after he click "Show database view" - although I don't know how hard is it to raise whole main windows in response.

Comment (jacek-izykowski) [inline src/ui/veles_mainwindow.cc]: Fixed.

File: src/ui/veles_mainwindow.cc (hunk 1 above)
Comment (jacek-izykowski) [inline src/ui/veles_mainwindow.cc]: Fixed.

**Pull request merged**